### DISCUSSION TITLE
Jeans

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#339] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/tirinhas/jeans/

**Comment by brunoaugusto** [comment]
Bem legal, apesar de um pouco exagerado :stuck\_out\_tongue:

**Comment by angellisy** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 20 de junho de 2022](<https://www.vish.blog.br/imagens/tirinhas/jeans/#comment_1160606074>):

> Bem legal, apesar de um pouco exagerado 😛

Isso é o que eu mais gosto nas *webcomics* da Cassandra. Isso e o fato de serem fáceis de traduzir, claro :sweat\_smile:
